src/components/Counter/Counter.jsx

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react';
+import Button from "../Button/Button"
+
+    function Counter({ setFruitCount, fruitCount }) {
+        return (
+            <>
+                <Button type="button" clickHandler={() => {if(fruitCount>0) setFruitCount(fruitCount - 1)}}>
```

Comment:
Ik had er zelf voor gekozen om de button te disable'en als `fruitCount` 0 is
```suggestion
                <Button type="button" clickHandler={() => setFruitCount(fruitCount - 1)} disabled={fruitCount === 0}>
```